Spanner/src/Batch/BatchClient.php

```diff
@@ -146,7 +146,7 @@ public function __construct(Operation $operation, $databaseName, array $options
     {
         $this->operation = $operation;
         $this->databaseName = $databaseName;
-        $this->databaseRole = isset($options['databaseRole']) ? $options['databaseRole'] : null;
+        $this->databaseRole = isset($options['databaseRole']) ? $options['databaseRole'] : "";
```

Comment:
Hi, I was looking at the implementation of Ruby here: https://github.com/googleapis/google-cloud-ruby/pull/19067/files#diff-7cd78f56f8d92f2ba49aaf812c3ea6fb55e4c4133f98f61e526201ac8df3e45eR55

It is nil there, Different languages do differently here?